### ISSUE TITLE
Clarify usage instructions

### ISSUE THREAD

**Issue by kevgathuku** [dcposch/webtorrent-remote#5]
Hello,
Thanks for setting this up. It seems to be a useful way of using webtorrent remotely.
However, after reading the instructions in the README I'm not able to figure out how exactly to use this package.
I would like to set up a remote webtorrent process on a server and be able to access it from a local client. So far, I haven't figured out where I should specify options such as the server URL so that I can connect to the remote webtorrent process, and where I should include that info in the client so that it knows to connect to the correct server.

I'm not sure if this is the correct place to ask this, but I would really appreciate some clarification.
Thanks in advance.

**Comment by feross** [comment]
@kevgathuku This package is very low-level. You, as the user, are responsible for getting commands from the client to the server. This makes this package super-flexible and useful in many situations (in an electron app, on a remote server, etc.)

Also, there is no command line interface (CLI) for the client, just a Node.js API. So you'll have to build your own CLI or GUI to control the remote process.

In short, when either client/server calls `send`, you need to get that data to the other side and call the `receive()` method with the data. That's how the client/server stay connected.
